cmd/dkvsrv/main.go

```diff
@@ -163,25 +164,27 @@ func main() {
 
 	switch srvrRole {
 	case noRole:
-		dkvSvc := master.NewStandaloneService(kvs, nil, br, dkvLogger, statsCli, regionInfo)
+		dkvSvc := master.NewStandaloneService(kvs, nil, br, dkvLogger, statsCli, regionInfo, healthCheckPeriod)
```

Comment:
Looks like we have lot of additional params, maybe we should group into a struct. Field like `dkvLogger, statsCli, regionInfo, healthCheckPeriod`

Reply:
Created serveropts